### ISSUE TITLE
"Dimensions could not be found." error

### ISSUE THREAD

**Issue by mackonsti** [covidpass-org/covidpass#57]
Hello @marvinsxtr when I tried the Greek localisation on my iOS phone running 14.7.0 and selecting GRAY colour for the exported Wallet item, I noticed an error "Dimensions could not be found". I am not sure if this was AFTER just the QR scan from camera OR the generation of the Wallet item, as it's on the bottom and I accidentally saw the error

Here is the visual output:

![IMG_0168](https://user-images.githubusercontent.com/1313613/126971506-2aeaf862-3820-499e-b2a0-3ed391c743ca.JPG)

Where would this come from, any idea? Could it be due to the colour scheme or the localisation? Thank you.

**Comment by marvinsxtr** [comment]
I tried with a Greek certificate but could not reproduce this. It may be an error with the camera view. I assume this error does not mean that the pass cannot be generated?

**Comment by mackonsti** [comment]
You are correct, the pass could be generated indeed. Let me try again tonight when I have some time, with the same QR I tried again earlier.

Please, this error @marvinsxtr where does it come from, at which stage of the process? Is it an error text that you created? Just to understand. Thanks.

**Comment by mackonsti** [comment]
**UPDATE:** The colour does not matter. Tried it with the same QR code as this morning, it seems that this error has to do with the camera detection of the QR itself. I managed to reproduce it 1 of 3 times by placing the QR code too low or too high on the live camera window...

When we tap to enable the camera, it is not immediately visible that we should scroll down further the window in order to access the full camera view, and center the QR code for better/faster detection. So if QR code is located to the top/bottom edge, this error could occur.

What would be good to understand is if this error comes from some API or the web-app itself, Marivn, so you can close the ticket. Thanks!

**Comment by marvinsxtr** [comment]
Now I am pretty sure that it comes from the QR code scanner/camera view. When an error is not translated, it means that we do not handle it explicitly. I may fix this together with some other changes in the future as this does not break the app.

**Comment by JensMertelmeyer** [comment]
Same for me, I cannot get it to work with two German certificates I got today. The message just flashes for a very short time on the bottom and is hard to notice.
What can I do to find out the cause?

**Comment by LukasAppleFan** [comment]
I had that error as well but I was still able to generate de QR Code. Living in France.

**Comment by marvinsxtr** [comment]
Fixed in #128
